go.mod

```diff
@@ -25,3 +25,31 @@ require (
 	k8s.io/klog v1.0.0
 	sigs.k8s.io/controller-runtime v0.8.2
 )
+
+replace k8s.io/api => k8s.io/api v0.25.4
+
+replace k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.25.4
+
+replace k8s.io/apimachinery => k8s.io/apimachinery v0.25.5-rc.0
+
+replace k8s.io/apiserver => k8s.io/apiserver v0.25.4
+
+replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.25.4
+
+replace k8s.io/client-go => k8s.io/client-go v0.25.4
```

Comment:
Instead of using replae directive here, this should be added in the require itself

Reply:
✔️